cocotb/drivers/__init__.py

```diff
@@ -130,7 +137,7 @@ def send(self, transaction, sync=True, **kwargs):
 
         Args:
             transaction (any): The transaction to be sent.
-            sync (bool,  optional): Synchronise the transfer by waiting for rising edge.
+            sync (bool, optional): Synchronise the transfer by waiting for rising edge.
```

Comment:
nit: "for _a_ rising edge" or "for _the next_ rising edge"

Reply:
Done.